include/internal/libspdm_common_lib.h

```diff
@@ -63,22 +63,22 @@ typedef struct {
     /* My Certificate */
     const void *local_cert_chain_provision[SPDM_MAX_SLOT_COUNT];
     size_t local_cert_chain_provision_size[SPDM_MAX_SLOT_COUNT];
-    /* My provisioned certificate (for slot_id - 0xFF, default 0)*/
-    uint8_t provisioned_slot_id;
```

Comment:
Why remove `provisioned_slot_id`?

Reply:
`provisioned_slot_id` was just for the synchronization from slot id 0xFF of the request message to the one of slots (default is slot 0) libspdm holding internally in its context. We were doing so because the raw public key of slot 0xFF was using certificate process routine.
From this PR the 0xFF slot id can be processed directly so the `provisioned_slot_id` is no longer needed.